### PR TITLE
Add contribution blocking level

### PR DESCRIPTION
**Pull request by itsibitzi**
Rename `AdBlockingLevel` to `BlockingLevel` and add contribution ask blocking.

### CODE REVIEW

**Review by akash1810 — COMMENTED** [review]
Can we freely rename fields in thrift, or do we need some sort of migration?

**Comment by itsibitzi** [comment]
My understanding is the thrift would still serialize anyway since it's based on slots, but might be wrong there. That being said we never used the previous version!

File: src/main/thrift/tag.thrift [hunk 1]
```diff
@@ -218,5 +218,8 @@ struct Tag {
     29: optional image.Image footballCrest;
 
     /** Ad blocking level used to block ads when a tag is applied to content */
-    30: optional AdBlockingLevel adBlockingLevel;
+    30: optional BlockingLevel adBlockingLevel;
+
+    /** Ad blocking level used to block ads when a tag is applied to content */
```
Comment (sihil) [inline src/main/thrift/tag.thrift]: Comment is identical to that on the field above.

File: version.sbt [hunk 2]
```diff
@@ -1 +1 @@
-version in ThisBuild := "2.5.0"
+version in ThisBuild := "2.6.0"
```
Comment (sihil) [inline version.sbt:1]: Typically this is bumped automatically by the release plugin and should not be touched in a PR.

**Review by sihil — CHANGES_REQUESTED** [review]
Couple of things to clarify.

**Comment by sihil** [comment]
The field hasn't been renamed - only the name of the enum. I'm not sure if that's a breaking change but understand why it's been made.

**Comment by itsibitzi** [comment]
Fixed the comments - is it ok to merge this? I'd prefer to roll this out ahead of the tag manager and flexible content changes

**Comment by sihil** [comment]
Sorry for missing this @itsibitzi. Was about to ask for further clarification as to why the version was being bumped in a PR. Just looked and saw this isn't using the release plugin which (for me) violates principle of least surprise.

**Review by sihil — APPROVED** [review]

**Comment by sihil** [comment]
Sorry for missing this @itsibitzi. Was about to ask for further clarification as to why the version was being bumped in a PR. Just looked and saw this isn't using the release plugin which (for me) violates principle of least surprise.

**Pull request merged**